interface/billing/print_daysheet_report_num1.php

```diff
@@ -68,1764 +68,1968 @@
+                                                <td class='text' width='250'>
+                                                </td>
+                                                <td class='text' width='125'>
+                                                </td>
+                                                <td class='text' width='175' height='5'>
+                                                   <hr class='font-weight-bold' noshade size='3'>
+                                                </td>
+                                                <td class='text' width='25'>
+                                                    <br />
+                                                </td>
+
+                                                <table class='table table-borderless'>
+                                                    <tr>
+                                                        <td class='text' width='25'>
+                                                        </td>
+                                                        <td class='text' width='250'>
+                                                        </td>
+                                                        <td class='text' width='125'>
+                                                        </td>
+                                                        <td class='text' width='250'>
+                                                        </td>
+                                                        <td class='text' width='125'>
+                                                        </td>
+                                                        <td class='text font-weight-bold' width='250'>
+                                                            <?php echo xlt("Actual Receipts") . ': ' . "(%1\$.2f)", text($provider_info['patref'][$i] + $provider_info['insref'][$i] + $provider_info['inspay'][$i] + $provider_info['patpay'][$i]); ?>
+                                                        </td>
+                                                        <td class='text' width='25'>
+                                                        </td>
+
+                                                        <table class='table table-borderless'>
+                                                            <tr>
+                                                                <td class='text' width='25'>
+                                                                </td>
+                                                                <td class='text' width='250'>
+                                                                </td>
+                                                                <td class='text' width='125'>
+                                                                </td>
+                                                                <td class='text' width='250'>
+                                                                </td>
```

Comment:
Hi @tywrenn  and @bradymiller 

I convert the print function to the pure HTML and found the complex nested table, looks like it didn't have </table> to close tag?

Reply:
That's odd. How do you even trigger this thing @bradymiller

Reply:
Yeah, maybe need @bradymiller, sir  guide us on how to trigger that page.

Reply:
Can get to this page via:
Fees->Billing Manager->End of Day Report